src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -97,7 +97,8 @@
 FATAL_ERROR_CHECK_FAILURE = re.compile(
     r'#\s+(Check failed: |RepresentationChangerError: node #\d+:)?(.*)')
 FATAL_ERROR_DCHECK_FAILURE = re.compile(r'#\s+(Debug check failed: )(.*)')
-FATAL_ERROR_REGEX = re.compile(r'#\s*Fatal error in (.*)')
+FATAL_ERROR_REGEX = re.compile(r'(^|#\s*)Fatal error(:| in) (.*)',
```

Comment:
I'm not sure i like conflating two conpletely different crash types here... I think it's better to be explicit about differences rather than trying generalise things here. Especially if we plan on refactoring this code in the future

Reply:
Can do, better to do two different regexes then. Also, that branch does "state.fatal_error_occurred = True", separating will help to not set it in golang change, also simplifies the if that will not be needed if we have two regexes.
```
if is_golang:
          state.crash_state = fatal_error_match.group(3) + '\n'
        else:
          state.crash_state = filter_stack_frame(fatal_error_match.group(3))
```

Reply:
Done, thanks!